internal/util/update/update_test.go

```diff
@@ -466,6 +466,57 @@ func TestCommand_Run_toTagRef(t *testing.T) {
 	}
 }
 
+// TestCommand_ResourceMerge_NonKRMUpdates tests if the local non KRM files are updated
+func TestCommand_ResourceMerge_NonKRMUpdates(t *testing.T) {
```

Comment:
Update the test to cover:

- File is added upstream (add)
- File is deleted upstream (delete)
- File is added locally (keep)
- File is deleted upstream and changed locally (error or delete)
- File is unchanged upstream, and changed locally (keep local copy)

Reply:
Leave local file untouched if it is added/modified